### ISSUE TITLE
Missing alert code

### ISSUE THREAD

**Issue by onkelfarmor** [jm-73/Indego#19]
Looks like the API is missing an alert code - suddenly got this message after restart of HA, but after deleting the alerts in the app, I don't get the error anymore and the component works again.

I had to different alerts, so I can't tell which was causing the error.

```
Error during setup of component indego
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/setup.py", line 172, in _async_setup_component
    component.setup, hass, processed_config  # type: ignore
  File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/config/custom_components/indego/__init__.py", line 54, in setup
    Mower()
  File "/config/custom_components/indego/__init__.py", line 130, in __init__
    IndegoAPI_Instance.AlertsDescription()
  File "/usr/local/lib/python3.7/site-packages/pyIndego/__init__.py", line 816, in AlertsDescription
    self._alert1_friendly_description = ALERT_ERROR_CODE[tmp_code]
KeyError: '1108'
```

**Comment by jm-73** [comment]
OK, it is probably by design. But I can see that I need to handle this in another way.

If you can trigger the alarm again, please give me the alert description so I can add it to the code.

**Comment by jm-73** [comment]
Will be fixed in next release!

**Comment by jm-73** [comment]
Fixed in release 0.7.1
